js-console/src/main/scala/org/typelevel/log4cats/console/ConsoleLogger.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats
+package console
+
+import cats._
+import cats.syntax.all._
+
+class ConsoleLogger[F[_]: Applicative: ConsoleF] extends SelfAwareStructuredLogger[F] {
+  override def trace(t: Throwable)(message: => String): F[Unit] = ConsoleF[F].debug(message, t)
```

Comment:
Wouldn't it be better if we create an underlying `ConsoleF` instance and use it anywhere within `ConsoleLogger`?

Reply:
Addressed in 4cc405b4.